### ISSUE TITLE
allowable use of wiki.openstreetmap.org

### ISSUE THREAD

**Issue by hbogner** [osm-cwg/tasks#17]
Sent to communications@

"Quick question re: allowable use of http://wiki.openstreetmap.org/?

OsmAnd is a very popular and broadly distributed open source mobile app (in Android, Linux, and the just released Apple IOS versions) that consumes and promotes the use of OSM data.

    Would it be O.K. to transfer and continue to build the design and user manual for osmAnd on wiki.openstreetmap.org?

osmAnd currently is migrating from Google to github as Google is shutting down its code repositories. osmAnd would greatly benefit from a standard MediaWiki site to host and evolve its documentation going forward.

The closer the integration with Wikipedia and OSM the better. A key feature of osmAnd is the ability to map WikiPedia articles with lat/long points easily on to a mobile map, so the technical relationship between Wikipedia, OSM, MediaWiki and osmAnd, is especially critical, on many levels, and likely to rapidly evolve going forward. With ever expanding use of cross-linked data the need for very clear communications becomes paramount.

Having osmAnd users arrive at the OSM site really helps drive active data end users to know more about the total nature of openstreetmap.org. Some of those people can likely be converted to OSM contributors over time.

User manuals are closely interconnected with useability, and software interface design. As OSM data continuously expands in granularity and greatly enhanced accuracy, osmAnd software will need to follow that evolution step-by-step. As data becomes richer osmAnd must evolve its user interface to simplify access to ever greater volumes of map data without overwhelming the user. Likewise, osmAnd is challenged to migrate its software code base to ever enhanced, ever more economical hardware platforms. Yet with all this expansion the standing user base and new users can't be overwhelmed with discontinuous complexity. Documentation and communications functions are very important in easing the path forward.

Having access to the standard OSM MediaWiki platform for its design and user documentation would be a real plus. Any reasonable OSM policy could be followed by osmAnd (such as title each MediaWiki article page beginning with "osmAnd - ", etc.)

I can't imagine the total usage here would place any burden on the standing OSM servers. It might involve a dozen or so new wiki contributors and perhaps, over time, a few hundred article pages, some with static graphics (screen shots, icons, etc.). The frequency of SQL commits would be quite low as documentation tends to last for a long while (vs. breaking news, etc). User documentation for software also tends to be very low interest as soon as the user is through the learning curve and knows how to get what they want. osmAnd developers (particularly the UI group) would benefit greatly by easy to access statistics on what features are not "intuitive", where users are hitting snags, whats clearly not operating as expected, etc. Its a "closed loop" feedback system.

OSM data modelers, and data architects might also benefit by having analytical feedback from heavy users. This might help illuminate structural changes to the data models that would allow for much more efficient data exportation from the OSM data servers.

Bottom line, the more streamlined and useful the software, the more the underlying data actually gets used. Everyone wins.

    Does the communications WG handle this type of inquiry? (If I'm way off, sorry about that, who should be contacted?)
    Can osmAnd just "try it", and if everyone hates it at least get assistance in exporting those new MediaWiki pages to a new, independent server?
    Who at OSM might be willing to provide some thoughtful input and guidance here?

Thanks!

Rick"

**Comment by harry-wood** [comment]
replied saying "YES"

also pointing to this page as a good discussion location: http://wiki.openstreetmap.org/wiki/Talk:OsmAnd
